skt/__init__.py

```diff
@@ -497,11 +520,19 @@ def __init__(self, path, basecfg, cfgtype=None, makeopts=None,
 
     def prepare(self, clean=True):
         if (clean):
-            args = self.defmakeargs + ["mrproper"]
+            args = self.makeopts + ["mrproper"]
```

Comment:
Do we need 'mrproper' any more if we have a fresh build area.  I think this was a legacy use when the same output build dir was being used.

Reply:
The `make mrproper` is required since `make rh-configs` and `make olddefconfig` will make the kernel source unclean. This will cause the `make targz-pkg` to fail with an error that says you must run `make mrproper` first.

Reply:
really?  That seems odd.  I just ran those commands on rhel-7 with no problem.  Is this leftover from output directory re-use?

Reply:
That could be the case. However, I know if you put the `.config` into the kernel source directory, `make olddefconfig`, and then copy the `.config` to the output directory, `make` will complain at you for not running `make mrproper`. The build will fail.

This may only show up when building kernels with the output outside of the source directory.

Reply:
Oh right, because the .config file is there or even worse all the include/autoconf stuff is there.  Yes the O= is the problem and is caused by the 'make rh-configs' cluttering up the local env.  
Ok, so I understand the problem now.  This is fine. I should probably file a separate issue to either comment that fact or fix it by using 'make O=<output> rh-configs'.  I just verified the latter works.